wafer/talks/models.py

```diff
@@ -47,6 +48,21 @@ def authors_help():
     ))
     return ' '.join(text)
 
+class TalkTypeManager(models.Manager):
+
+    def open_for_submission(self, plus=models.Q(pk__lt=0)):
```

Comment:
I would remove the `plus` parameter and simply allow the caller to do their own `| Q(pk=initial) `.

Reply:
done